src/TextField/TextField.js

```diff
@@ -115,6 +126,11 @@ export default class TextField extends Component {
           disabled={disabled}
           {...inputProps}
         />
+        {helperText && (
```

Comment:
I would rather see the following pattern. For instance try with `helperText = 0`, that won't render correctly.
```js
helperText ? : null
```